Source/WebCore/workers/service/server/SWServerJobQueue.cpp

```diff
@@ -116,7 +116,9 @@ void SWServerJobQueue::scriptFetchFinished(const ServiceWorkerJobDataIdentifier&
         return;
     }
 
-    m_server.updateWorker(job.identifier(), *registration, job.scriptURL, result.script, result.certificateInfo, result.contentSecurityPolicy, result.crossOriginEmbedderPolicy, result.referrerPolicy, job.workerType, { }, job.serviceWorkerPageIdentifier());
+    auto topRegistrableDomain = RegistrableDomain::uncheckedCreateFromHost(m_registrationKey.topOrigin().host);
+
+    m_server.updateWorker(topRegistrableDomain, job.identifier(), *registration, job.scriptURL, result.script, result.certificateInfo, result.contentSecurityPolicy, result.crossOriginEmbedderPolicy, result.referrerPolicy, job.workerType, { }, job.serviceWorkerPageIdentifier());
```

Comment:
It does not seem necessary to pass the registrable domain since we pass registration which has the top origin in its key as well.